### DISCUSSION TITLE
难道你们不二模吗？

### DISCUSSION THREAD

**Discussion by langningchen** [XMOJ-Script-dev/XMOJ-Script#500] [General]
马上就二模了，在二模的时候改代码，也不早点改，复习的怎么样？ @boomzero @PythonSmall-Q

**Comment by boomzero** [comment]
怎么了

**Comment by langningchen** [comment]
就是感觉好奇怪

**Comment by boomzero** [comment]
Ps: 闵行二模下周二

**Comment by langningchen** [comment]
我们刚考完语文 《有我在》

**Comment by langningchen** [comment]
你加油

**Comment by langningchen** [comment]
综合也考完了 😢

**Comment by PythonSmall-Q** [comment]
黄浦考完啦！！！心情很好

**Comment by langningchen** [comment]
@PythonSmall-Q 怎么样？考得很好？

**Comment by langningchen** [comment]
#500 🎉

**Comment by PythonSmall-Q** [comment]
github education尝试apply没反应

**Reply by langningchen** [reply]
@PythonSmall-Q 你在二模的讨论里说这个？

**Comment by PythonSmall-Q** [comment]
小明的OJ增强脚本健康检查 - Service Monitoring (xmoj-bbs.tech)

**Reply by PythonSmall-Q** [reply]
@langningchen

**Reply by PythonSmall-Q** [reply]
https://status.xmoj-bbs.tech @langningchen 去看看吧

**Reply by langningchen** [reply]
@PythonSmall-Q No thanks

**Comment by boomzero** [comment]
@PythonSmall-Q ~看了一下黄浦卷子, 感觉比杨浦好多了~

**Comment by boomzero** [comment]
@PythonSmall-Q 怎么样？

为什么我觉得你们道法第二大题在说数字教育很好然后第三大题又说数字教育不好……

**Reply by langningchen** [reply]
有道理

**Comment by boomzero** [comment]
@langningchen @PythonSmall-Q 周末要做黄浦+浦东二模

**Reply by langningchen** [reply]
@boomzero 哈哈哈，我们已经做过黄浦卷了

**Reply by PythonSmall-Q** [reply]
@langningchen 黄浦卷感觉怎么样

**Reply by boomzero** [reply]
比闵行简单

**Reply by langningchen** [reply]
@PythonSmall-Q 还行

**Comment by langningchen** [comment]
我们考完了，彻底废了

**Comment by boomzero** [comment]
明天体育中考游泳

**Reply by langningchen** [reply]
我们下周二😎 @PythonSmall-Q 你呢？

**Reply by PythonSmall-Q** [reply]
今天下午除了游泳之外的

**Reply by PythonSmall-Q** [reply]
游泳周一

**Comment by boomzero** [comment]
你们怎么工作日考啊……我们都是周末(游泳)

**Comment by boomzero** [comment]
?

**Comment by boomzero** [comment]
<img width="1920" alt="image" src="https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/85378277/89fa5ff8-8d33-444f-b8b9-5253ef17a9e3">

**Comment by PythonSmall-Q** [comment]
体育中考满分了呦吼 @langningchen  @boomzero

**Comment by boomzero** [comment]
游泳满分了

**Reply by PythonSmall-Q** [reply]
加油！

**Reply by boomzero** [reply]
@PythonSmall-Q 你们18题什么鬼啊, 我算出来数字的非常奇妙

**Reply by boomzero** [reply]
还有第三题...


<img width="312" alt="image" src="https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/85378277/58522940-146b-4306-bf46-03527f1cafd3">

**Reply by boomzero** [reply]
ps: source code: https://www.luogu.com.cn/paste/9k7qhf2c

**Reply by PythonSmall-Q** [reply]
第三题D

**Comment by PythonSmall-Q** [comment]
7：8，我对了

发件人: Zhu Chenrui
发送时间: 2024年4月14日 10:37:22
主题: Re: [XMOJ-Script-dev/XMOJ-Script] 难道你们不二模吗？ (Discussion #500)

@PythonSmall-Q 你们18题什么鬼啊, 我算出来数字的非常奇妙

**Comment by langningchen** [comment]
@boomzero @PythonSmall-Q 告诉你们一个很悲伤的消息，我体育校外只有14分，总分29 😭

**Reply by langningchen** [reply]
全扣在了篮球上

**Comment by PythonSmall-Q** [comment]
你还是篮球！！！

**Comment by boomzero** [comment]
@langningchen 数学24把D在x轴上看成了D在y轴上, -9...

问题是那题还很简单, 我还做出来了!!!...

**Comment by boomzero** [comment]
二模废了...

**Comment by boomzero** [comment]
一下子送了9分....

**Comment by langningchen** [comment]
等成绩出来了我们再看看情况

**Comment by langningchen** [comment]
@PythonSmall-Q @boomzero 你们报暑假班吗？报哪一个？提高A/B？

**Comment by boomzero** [comment]
显然B, A要CSP-S >250pts

**Comment by PythonSmall-Q** [comment]
我目前还没有报班的想法

**Comment by langningchen** [comment]
> 显然B, A要CSP-S >250pts

@boomzero 很好，我应该也是B，期待暑假班与你再次相遇

**Comment by PythonSmall-Q** [comment]
别忘了宣传XMOJ-Script

**Comment by langningchen** [comment]
@PythonSmall-Q @boomzero 我们出二模成绩了

**Comment by langningchen** [comment]
@PythonSmall-Q @boomzero 

告诉你们一个很遗憾的消息，我今天放学回家的路上决定完全彻底停止对 `XMOJ-Script` 的支持，包括不再创建新的 Issues ，审阅 PR ，提交 Commit 等，并且也会逐步停止使用 `XMOJ-Script` ，因为现在这个项目已经违背了我原始 `XMOJ-Script` 的初衷：致力于帮助用户在算法竞赛中提升成绩。现在 `自动提交AC代码` ， `讨论区 中与题目无关的讨论内容` 等功能都不符合这个初衷，并且代码很乱。

我可能会创建一个新的项目，名为 `XMOJ (小美的OJ)` ，他是一个 `XMOJ (小明的OJ)` 的第三方 ~客户端~ 网站，并且我正在尝试购买一个新的域名 `langningchen.com`。

~如果我现在还是管理员，我会让 `XMOJ-Script` 的生命周期停止于 2024/06/30~

**Comment by boomzero** [comment]
其他的明天回复， 就是说一下，this is probably not a good idea，因为不能在高老师的网络环境下访问

**Reply by langningchen** [reply]
我知道，所以我这个软件可以运行本地服务器，再说xmoj的讨论区不是也不能访问吗？

**Comment by boomzero** [comment]
不如直接写UI，我可能会写一个，以前我试过，但是用的SwiftUI在 2024年4月24日，18:02，Langning Chen ***@***.***> 写道：﻿
我知道，所以我这个软件可以运行本地服务器，再说xmoj的讨论区不是也不能访问吗？

**Comment by boomzero** [comment]
显然那个不是核心功能发自我的 iPhone在 2024年4月24日，18:02，Langning Chen ***@***.***> 写道：﻿
我知道，所以我这个软件可以运行本地服务器，再说xmoj的讨论区不是也不能访问吗？

**Comment by boomzero** [comment]
指 native GUI发自我的 iPhone在 2024年4月24日，18:11，陈琦 ***@***.***> 写道：﻿不如直接写UI，我可能会写一个，以前我试过，但是用的SwiftUI在 2024年4月24日，18:02，Langning Chen ***@***.***> 写道：﻿
我知道，所以我这个软件可以运行本地服务器，再说xmoj的讨论区不是也不能访问吗？

**Comment by boomzero** [comment]
flutter发自我的 iPhone在 2024年4月24日，18:11，陈琦 ***@***.***> 写道：﻿不如直接写UI，我可能会写一个，以前我试过，但是用的SwiftUI在 2024年4月24日，18:02，Langning Chen ***@***.***> 写道：﻿
我知道，所以我这个软件可以运行本地服务器，再说xmoj的讨论区不是也不能访问吗？

**Comment by boomzero** [comment]
Question:为什么 oierdb 里没有swx发自我的 iPhone在 2024年4月24日，18:11，陈琦 ***@***.***> 写道：﻿不如直接写UI，我可能会写一个，以前我试过，但是用的SwiftUI在 2024年4月24日，18:02，Langning Chen ***@***.***> 写道：﻿
我知道，所以我这个软件可以运行本地服务器，再说xmoj的讨论区不是也不能访问吗？

**Comment by langningchen** [comment]
@boomzero Emm... 你Email回复的格式好乱啊……

**Comment by boomzero** [comment]
GitHub 的问题啊发自我的 iPhone在 2024年4月24日，21:54，Langning Chen ***@***.***> 写道：﻿
@boomzero Emm... 你Email回复的格式好乱啊……

**Comment by langningchen** [comment]
@boomzero @PythonSmall-Q 你们做什么试运行试卷了吗？

**Reply by PythonSmall-Q** [reply]
做了

**Comment by boomzero** [comment]
做了呀, 作文11

**Comment by boomzero** [comment]
@langningchen 其他的只扣1.5, 但是作文11

**Comment by boomzero** [comment]
废了

**Comment by langningchen** [comment]
真厉害

**Comment by langningchen** [comment]
我作文扣了两分，但是前面扣了一堆

**Comment by boomzero** [comment]
...

**Comment by boomzero** [comment]
英语138!!!

**Comment by boomzero** [comment]
...

**Comment by boomzero** [comment]
听说8.5

**Comment by langningchen** [comment]
太有实力了
